Gemfile.lock

```diff
@@ -58,4 +58,4 @@ DEPENDENCIES
   jekyll (~> 3.8.5)
 
 BUNDLED WITH
-   2.1.4
+   2.2.15
```

Comment:
This change was not intended, I will revert.

Reply:
```suggestion
   2.1.4
```

Reply:
Thanks @awvwgk